tests/ui_tests/test_actions_page.py

```diff
@@ -0,0 +1,55 @@
+import time
+
+import allure
+import pytest
+from adcm_client.base import ObjectNotFound
+from adcm_client.objects import ADCMClient
+from adcm_pytest_plugin import utils
+
+from tests.ui_tests.app.actions_page import ActionPage
+# pylint: disable=W0621
+
+
+@allure.step('Create cluster')
+@pytest.fixture()
+def cluster(sdk_client_fs: ADCMClient):
+    bundle_dir = utils.get_data_dir(__file__, "nothing_happens")
+    bundle = sdk_client_fs.upload_from_fs(bundle_dir)
+    return bundle.cluster_create(utils.random_string())
+
+
+@allure.step('Open ADCM tab Action')
+@pytest.fixture()
+def action_page(app_fs, login_to_adcm, cluster):
+    return ActionPage(app_fs.driver, f"{app_fs.adcm.url}/cluster/{cluster.cluster_id}/action")
+
+
+def wait_for_task(cluster, interval=1, timeout=30):
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            return cluster.action(name="test_action").task()
```

Comment:
Not sure if it's a good idea to hardcode action name

Reply:
done